_data/sitetext.yml

```diff
@@ -95,16 +96,18 @@ de: &DEFAULT_DE
     # left is the default
     #start_align: "left"
     events:
-      - title: "Stand Gründung der Genossenschaft :"
-        year: "2022"   
-        desc: "• Die Gründung, die Besetzung der Ämter sowie die Suche nach einer geeigneten Immobilie wird parallel durchgeführt.<br>
-           • Weiterhin sind wir auf jede Art von Unterstützung angewiesen!<br>
-           Sollten Sie also Interesse an einer aktiven Mitarbeit haben, oder jemanden kennen der eine Immobilie anbieten kann, dann melden Sie sich gerne per Mail unter info@unverpackt-mucnord.de"
-        align: left
+      - title: "Wir haben offiziell die Genossenschaft gegründet!"
+        year: "07.03.2022"
+        desc: "Im Stadtteilladen Moosach haben wir mit Hilfe von Frau Bahar und Herrn Max Riedl der Genossenschaftsverband Bayern 
+          die Genossenschaft *Unverpackt Laden München Nord eG* gegründet. Jetzt sind wir bereit für die nächste Schritte!<br>
+           Weiterhin sind wir auf jede Art von Unterstützung angewiesen!<br>
+           Sollten Sie also Interesse an einer aktiven Mitarbeit haben, oder jemanden kennen, der eine Immobilie anbieten kann, dann 
+           melden Sie sich gerne per Mail unter [info@unverpackt-mucnord.de](mailto:info@unverpackt-mucnord.de)"
         image: ../../assets/img/timeline/Faust.jpg
+        align: left
         alt:
       - title: "Gründungstreffen geplant"
-        year: "07.03.22, 19.30 Uhr"
+        year: "03.22, 19.30 Uhr"
```

Comment:
Should it just be the month without day?
Maybe "März 22" would fit better then? :shrug: